assets/src/date/test/isYesterday.js

```diff
@@ -0,0 +1,48 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * External dependencies
+ */
+import moment from 'moment-timezone';
+
+/**
+ * Internal dependencies
+ */
+import { isYesterday } from '../';
+
+describe('date/isYesterday', () => {
+  it('should return true that today minus 1 day is yesterday', () => {
+    const dateString = moment().subtract(1, 'days');
```

Comment:
It's always a bit risky to rely on the actual current date due to timezones, tests running over midnight, DST changes, etc.

We should probably use `mockdate` consistently everywhere.

Reply:
oh good catch!